src/test/java/linenux/command/RemindCommandTest.java

```diff
@@ -0,0 +1,256 @@
+package linenux.command;
+
+import static linenux.helpers.Assert.assertChangeBy;
+import static linenux.helpers.Assert.assertNoChange;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import linenux.command.parser.ReminderArgumentParser;
+import linenux.command.result.CommandResult;
+import linenux.model.Reminder;
+import linenux.model.Schedule;
+import linenux.model.Task;
+
+/**
+ * JUnit test for remind command.
+ */
+public class RemindCommandTest {
+	private Schedule schedule;
+	private RemindCommand remindCommand;
+
+	@Before
+	public void setupRemindCommand() {
+		ArrayList<Task> tasks = new ArrayList<>();
+		tasks.add(new Task("Todo1"));
+		tasks.add(new Task("Todo2"));
+		tasks.add(new Task("Deadline1", LocalDateTime.of(2016, 1, 1, 1, 0)));
+		tasks.add(new Task("Deadline2", LocalDateTime.of(2016, 1, 1, 13, 0)));
+		tasks.add(new Task("Event1", LocalDateTime.of(2016, 1, 1, 1, 0), LocalDateTime.of(2016, 1, 1, 13, 0)));
+		tasks.add(new Task("Event2", LocalDateTime.of(2016, 1, 1, 13, 0), LocalDateTime.of(2016, 1, 1, 23, 0)));
```

Comment:
Can you keep track of each instance of `Task` in an instance variable? The current setup is too fragile. For e.g., if I add

```java
tasks.add(new Task("Todo0"));
```

before line 31, more than half of the test cases here will fail.